jwst/resample/resample_spec_step.py

```diff
@@ -27,6 +28,20 @@ class ResampleSpecStep(ResampleStep):
 
     class_alias = "resample_spec"
 
+    spec = """
+        pixfrac = float(min=0.0, max=1.0, default=1.0)  # Pixel shrinkage factor
+        kernel = option('square', 'point', default='square')  # Flux distribution kernel
+        fillval = string(default='NAN')  # Output value for pixels with no weight or flux
+        weight_type = option('ivm', 'exptime', None, default='ivm')  # Input image weighting type
+        output_shape = int_list(min=2, max=2, default=None)  # [x, y] order
```

Comment:
I haven't seen `int_list()` before but I assume the min, max is the length of the list?

Reply:
Yes, that's right.  In case you haven't seen them, I just came across these useful docs for spec types:
https://jwst-pipeline.readthedocs.io/en/latest/jwst/stpipe/devel_step.html#configspec-types